io.sourceforge.pysolfc.PySolFC.json

```diff
@@ -0,0 +1,127 @@
+{
+  "app-id": "io.sourceforge.pysolfc.PySolFC",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "pysol",
+  "cleanup": [
+    "/bin/sqlite3_analyzer",
+    "/bin/tclsh8.6",
+    "/bin/wish8.6",
+    "/include",
+    "*.a",
+    "*.la",
+    "/lib/*Config.sh",
+    "/lib/cmake",
+    "/lib/pkgconfig",
+    "/lib64/pkgconfig",
+    "/man",
+    "/share/aclocal",
+    "/share/info",
+    "/share/man"
+  ],
+  "rename-icon": "pysol",
+  "rename-desktop-file": "pysol.desktop",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--persist=.PySolFC"
+  ],
+  "build-options": {
+      "env": {
+          "PORTMIDI_INC_PORTTIME": "1",
+          "PYGAME_EXTRA_BASE": "/app"
+      }
+  },
+  "modules": [
+    "tkinter.json",
+    "pygame2_extra_deps.json",
+    "python3-modules.json",
+    {
+      "name": "PySolFC-Cardsets",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir -p /app/share/PySolFC/cardsets/",
+        "cp -r * /app/share/PySolFC/cardsets/"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/shlomif/PySolFC-Cardsets",
+          "commit": "db74d748556b167052da9d6bcd7cc6b461351ff8",
+          "tag": "2.1",
+          "x-checker-data": {
+            "type": "git",
+            "tag-pattern": "^([\\d.]+)$"
+          }
+        }
+      ]
+    },
+    {
+      "name": "PySol-Music",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm644 data/music/* -t /app/share/PySolFC/music/"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/shlomif/pysol-music",
+          "commit": "108d6dcbb931f2b709f5a1590ea7ea6190f98d9e",
+          "tag": "4.50",
+          "x-checker-data": {
+              "type": "git",
+              "tag-pattern": "^([\\d.]+)$"
+          }
+        }
+      ]
+    },
+    "solvers.json",
+    {
+      "name": "PySolFC",
+      "buildsystem": "simple",
+      "build-commands": [
+        "make rules",
+        "make mo",
+        "ln -s data/images images",
+        "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} . --no-build-isolation",
+        "install -Dm755 pysol -t /app/bin/",
+        "install -Dm644 io.sourceforge.pysolfc.PySolFC.appdata.xml -t /app/share/metainfo/"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/shlomif/PySolFC",
+          "commit": "3879a9196b425b30b0d69caffb8988d11e6939b6",
+          "tag": "pysolfc-2.18.0",
+          "x-checker-data": {
+              "type": "git",
+              "tag-pattern": "^pysolfc-([\\d.]+)$",
+              "is-main-source": true
+          }
+        },
+        {
+          "type": "patch",
+          "path": "0001-use-wrapper-in-desktop-file.patch"
```

Comment:
you can just use `destkop-file-edit` instead of adding a patch.

Reply:
Huh. Learn something new every day.

I'll change that.